lib/matplotlib/ticker.py

```diff
@@ -1426,9 +1423,9 @@ def set_params(self, **kwargs):
         if 'integer' in kwargs:
             self._integer = kwargs['integer']
         if self._integer:
-            self._steps = [n for n in self._steps if divmod(n, 1)[1] < 0.001]
+            self._steps = [n for n in self._steps if _divmod(n, 1)[1] < 0.001]
 
-    def bin_boundaries(self, vmin, vmax):
```

Comment:
A bug fix (handling floating point better) would not in itself require a name change or deprecation, but I'm perfectly happy with the new private name.  To be on the safe side, a deprecation cycle of the old name would be a good idea.  (I thought it was being used elsewhere, but I haven't found any such instances.)